karma.conf.js

```diff
@@ -4,18 +4,30 @@ if (!process.env.TRAVIS){
     process.env.CHROME_BIN = require('puppeteer').executablePath();
 }
 
+function getTestFiles(){
+    switch (process.env.BROWSER_BUNDLE_TEST){
+        case 'publishedDist': return ["packages/web3/dist/web3.min.js", "test/e2e.minified.js"]
+        case 'gitRepoDist':   return ["dist/web3.min.js", "test/e2e.minified.js"]
+        default:              return ["test/**/e2e*.js"]
+    }
+}
+
+// Only loads preprocessor for the logic unit tests
+function getPreprocessors(){
+    if (!process.env.BROWSER_BUNDLE_TEST){
+        return { 'test/**/e2e*.js': [ 'browserify' ] }
+    }
+}
+
 module.exports = function (config) {
     var configuration = {
         frameworks: [
             'mocha',
             'browserify'
         ],
-        files: [
-            'test/**/e2e*.js'
-        ],
-        preprocessors: {
-            'test/**/e2e*.js': [ 'browserify' ]
-        },
+        files: getTestFiles(),
+        preprocessors: getPreprocessors(),
+        failOnEmptyTestSuite: false,
```

Comment:
Why is ``failOnEmptyTestSuite`` defined as ``false`` required? 🤔

Reply:
No! Good catch.

Reply:
Removed in 331b01176fc0c4f01bd066f172d0b303f84b8ab9